src/partials/10_header.html

```diff
@@ -56,6 +56,14 @@
           <form action="" class="search-form">
             <div class="category">
               <i class="fas fa-list-ul"></i>
+              <select name="" id="" class="ddProducts">
```

Comment:
Nie wiem czemu mają służyć te puste atrybuty `name="" id=""`, widzę, że jest ich więcej, można to będzie usunąć w oddzielnym PR, gdyby zabrakło tasków

Reply:
poprawione